### DISCUSSION TITLE
How do I switch to a lane in a different platform?

### DISCUSSION THREAD

**Discussion by kueda** [fastlane/fastlane#20410] [Q&A]
So far the only way I've found is this:

```ruby
platform :android do
  lane :build do
    puts "android foo lane"
  end
end

platform :ios do
  lane :foo do
    puts "ios foo lane"
  end
end

lane :foo do
  Fastlane::LaneManager.cruise_lane "ios", "foo"
  Fastlane::LaneManager.cruise_lane "android", "foo"
end
```

`Fastlane::LaneManager.cruise_lane` isn't documented so it's probably the wrong way to do that, right?

**Comment by slavdok** [comment]
I would like this too.

I've tried the `cruise_lane` approach, however doing that starts execution ALL the way from the top, including `before_all` being called again
